CodeEdit/Features/CEWorkspace/Views/Pages/TasksCEWorkspaceSettingsView.swift

```diff
@@ -0,0 +1,82 @@
+//
+//  TasksCEWorkspaceSettingsView.swift
+//  CodeEdit
+//
+//  Created by Axel Martinez on 27/3/24.
+//
+
+import SwiftUI
+
+struct TasksCEWorkspaceSettingsView: View {
+    let workspace: WorkspaceDocument
+
+    // TODO: Separate Project Settings from Task Settings
+    @Binding var projectSettings: CEWorkspaceSettingsData.ProjectSettings
+    @Binding var settings: CEWorkspaceSettingsData.TasksSettings
+
+    @State private var selectedTaskId: UUID?
+    @State private var addTaskSheetPresented: Bool = false
+
+    var body: some View {
+        SettingsForm {
+            Section {
+                TextField(text: $projectSettings.projectName) {
+                    Text("Name")
+                }
```

Comment:
```suggestion
                TextField("Name", text: $projectSettings.projectName)
```

Reply:
Is there a good reason to go with the 3-line version?

Reply:
I'll update it on the next commit.